src/app/components/sidebar/sidebar.component.ts

```diff
@@ -64,6 +50,9 @@ export class SidebarComponent implements OnDestroy {
     }
 
     //TODO (Mate): make this simpler
+    //@todo: there are bugs when working with multiple tools at once:
+    // 1. opening tools will give the correct initial state (no sidebar)
+    // 2. sidebar state will synchronize across all open tools
```

Comment:
@matesk8, this seems to be because the open and close instructions are going through the global eventHub, so they're propagated to all instances of sidebar. The reason why it works as expected when opening is because the sidebar is initialized in the right state.